experimental/regression_suite/ireers/fixtures.py

```diff
@@ -73,7 +75,7 @@ def iree_run_module(vmfb: Artifact, *, device, function, args: Sequence[str] = (
     exec_args.extend(args)
     print("**************************************************************")
     print("Exec:", " ".join(exec_args))
-    subprocess.check_call(exec_args, cwd=vmfb.group.directory)
+    subprocess.run(exec_args, check=True, capture_output=True, cwd=vmfb.group.directory)
```

Comment:
@stellaraccident do you have opinions on usability here? We can keep an eye on these jobs if/when they fail to see what other output or formatting would help. I was just making some tests fail locally to see that everything was working and noticed the lack of stdout/stderr output (`check_call` just sends them to the void and only checks the result code).